tensorrt_llm/_torch/pyexecutor/model_engine.py

```diff
@@ -1759,3 +1776,14 @@ def _init_userbuffers(self, hidden_size):
                                           self.mapping.gpus_per_node,
                                           hidden_size * self.max_num_tokens * 2)
         return True
+
+    def load_weights_from_target_model(self,
```

Comment:
Where is the method called?

Reply:
See `py_executor_creator.py:L98`. I'm not a huge fan of doing this but unfortunately the eagle checkpoint format that we're using sometimes is missing `embed_tokens` because it's meant to be shared with the target model.